plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1455,15 +1459,15 @@ public ColumnHandle getUpdateRowIdColumnHandle(ConnectorSession session, Connect
 
         Type rowIdType;
         if (unmodifiedColumns.isEmpty()) {
-            rowIdType = RowType.rowType(RowType.field(BIGINT));
+            rowIdType = RowType.rowType(field(BIGINT));
         }
         else {
-            List<RowType.Field> unmodifiedColumnFields = unmodifiedColumns.stream()
-                    .map(columnMetadata -> RowType.field(columnMetadata.getName(), columnMetadata.getType()))
+            List<Field> unmodifiedColumnFields = unmodifiedColumns.stream()
```

Comment:
I will revert this but thb I don't see a reason why this should be a separate commit. In this particular line it doesn't matter but I mainly imported it because I prefered:
```
Type type = RowType.rowType(rowIdFields.build().toArray(new Field[0]));
```
to
```
Type type = RowType.rowType(rowIdFields.build().toArray(new RowType.Field[0]));
```
and this is a new line coming from this change so couldn't be a part of different commit.